tasks/create-advisory-internal-request/tests/test-create-advisory-internal-request.yaml

```diff
@@ -179,3 +185,6 @@ spec:
                 echo "InternalRequest has the wrong config_map_name parameter"
                 exit 1
               fi
+
+              echo Test that the advisory_url result was properly set
+              test "$(echo $(params.advisory_url))" == "Advisory URL: https://github.com/org/repo/advisory"
```

Comment:
This seems wrong. I believe the result shouldn't have the "Advisory URL:" prefix in it - at least based on the line that saves the result in the task: `echo -n "Advisory URL: $(echo ${results} | jq -r '.advisory_url')" | tee $(results.advisory_url.path)`

Reply:
I'm not following. The line you posted even shows it echoing `Advisory URL: | tee results.advisory_url.path` of course the result will have Advisory URL in it

Reply:
Sorry, I misread. I don't know how that happened since I specifically looked at it several times to see if the `tee` applies to just the url or the whole string 🤦

So then my question is: Shouldn't we just save the actual url as the result (excluding "Advisory URL: ")? Even the result description says "The advisory url" so I would definitely expect just the url.

Reply:
I am cool with that. Will update